### ISSUE TITLE
Calculating MIOU on the evaluation set

### ISSUE THREAD

**Issue by SujithSash** [milesial/Pytorch-UNet#423]
Is there a way we could calculate MIOU on the validation set set at the time of training? Thanks in advance!

**Comment by FRC1221** [comment]
have you resolved it?
